src/main/webapp/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Air Quality Map</title>
+    <link rel="stylesheet" href="style.css">
+
+  </head>
+  <body>
+    <!-- Sidebar -->
+    <div class="sidenav">
+      <div class="topnav"> 
+        <div class="search-container">
+          <form action="/action_page.php">
+            <input type="text" placeholder="Search a location.." name="search">
+            <button type="submit" class="button Search" id="submit">Search</button>
+          </form>
+        </div>
+        <button class="button Search">Search</button>
+        <!-- trying button class vs id -->
```

Comment:
This comment is necessary for your development, but pls remove it when you try to merge this code to code base